homeassistant/components/sleepiq/manifest.json

```diff
@@ -11,5 +11,5 @@
   "documentation": "https://www.home-assistant.io/integrations/sleepiq",
   "iot_class": "cloud_polling",
   "loggers": ["asyncsleepiq"],
-  "requirements": ["asyncsleepiq==1.5.2"]
+  "requirements": ["asyncsleepiq==1.5.3"]
```

Comment:
Please split the dependency bump into a new PR and keep this one in draft until the other one is merged.
We don't allow adding new features and a dependency bump in the same PR. The reason behind is that the PRs get smaller and more important we can include the dependency bump in patch releases to fix bugs if needed

Reply:
@edenhaus understood on the don't-ping-members request.

On the dependency bump, I have removed that commit. It was not strictly necessary for the functionality that this PR is implementing.